network/src/noise_wrapper/mod.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate implements wrappers around our [Noise][noise] implementation.
+//! Noise is a protocol framework to encrypt and authentication connections.
+//! We use Noise to secure connections between peers in Libra.
+//! Specifically, we use the [Noise IK][ik] handshake which is a one round-trip protocol
+//! (the client sends one message, then the server responds).
+//! For more information about Noise and our implementation, refer to the [crypto] crate.
```

Comment:
I missed the part where the [crypto] crate (are you referring to `libra_crypto`?) contained information about Noise?

Reply:
yeah libra_crypto contains our implementation of noise

Reply:
1. `[crypto]`  will not doc-link to `libra_crypto`
2. how does your reader interpret the word `Noise` to refer to "Libra's single-pattern implementation of Noise"?